scripts/beastConfig.cmake.in

```diff
@@ -0,0 +1,31 @@
+# ===================================================================================
+#  The beast CMake configuration file
+#
+#             ** File generated automatically, do not modify **
```

Comment:
yes, this this is template to generate CMake helper that will make CMake find Beast easily. It goes to repository. Similar other Boost library: https://github.com/boostorg/hana/tree/master/cmake